pkg/util/netops/netops.go

```diff
@@ -78,16 +91,16 @@ func GetNS() (*NS, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to get a netlink handle for the current network namespace: %w", err)
 	}
-	ns := &NS{
+	ns := &namespace{
 		nsHandle: nsHandle,
 		handle:   handle,
-		Path:     path,
+		path:     path,
 	}
 	return ns, nil
 }
 
-// NewNamedNS returns an NS of the current network namespace
-func NewNamedNS(name string) (ns *NS, err error) {
+// NewNamespace returns a new named network namespace
```

Comment:
How about CreateNewNamespace ?

Reply:
I renamed this function to `CreateNamedNamespace`.

The named namespace is only used in a single location of the test code. To simplify the interface, I changed the function not to return a created Namespace object, and instead returns a path of the created namespace. So, the created namespace needs to be opened using `OpenNamespace` later.

I also removed  the corresponding `Delete` function in the Namespace interface, and added `DeleteNamedNamespace` as a top level function of this package.